main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
Reviewing the readme the project is requesting to have the user loop through the mad lib generator 10 times. Consider this code written below instead. 

```suggestion
while (loop < 11):
```